Solutions/Corvus.Monitoring.AspnetCore.Mvc.Specs/Features/ControllerOperations.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: Controller Operations
```

Comment:
Why this name? Isn't this all about observable actions?

Reply:
Updated the file and feature name.